### DISCUSSION TITLE
How to get text without \n ?

### DISCUSSION THREAD

**Discussion by romanboiko-softserve** [html-to-text/node-html-to-text#266] [Q&A] [answered]
Hi,
I'm using the convert method to transform strings like this: `"Item 1 - Gift wrapping: Yes&nbsp;<br />"` and getting in result strings like this: `"Item 1 - Gift wrapping: Yes \n".`
Is there a way to get the string without "\n"?

**Comment by KillyMXI** [comment, marked as answer]
This example can have multiple interpretations.
What exactly is the goal?

- remove only trailing whitespace and linebreaks - you can use [`trimEnd()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/trimEnd) after conversion;
- ignore all `<br />` tags - use `selectors: [ { selector: 'br', format: 'skip' } ]`.

**Reply by romanboiko-softserve** [reply]
I needed to get a result like this: `"Item 1 - Gift wrapping: Yes"`. 
trimEnd() helped, thanks.
I just thought there is a build in method for getting the format I needed.
